wasm/module_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 The go-interpreter Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package wasm_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/go-interpreter/wagon/wasm"
+)
+
+func TestReadModule(t *testing.T) {
+	fnames, err := filepath.Glob("testdata/*wasm")
```

Comment:
Maybe `filepath.Glob("testdata/*.wasm")` to make it more clear you're looking for all files with `.wasm` extension.

Also, I'm not sure how `filepath.Glob` operates on Windows and whether it'll work with forward slash separators. But I'm okay if you don't think it's worth dealing with at this stage.